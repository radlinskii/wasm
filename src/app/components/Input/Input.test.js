import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import InputComponent from './Input';

describe('InputComponent', () => {
    test(`Should match snapshot`, () => {
        const tree = ReactTestRenderer.create(<InputComponent />);

        expect(tree).toMatchSnapshot();
    });
});
