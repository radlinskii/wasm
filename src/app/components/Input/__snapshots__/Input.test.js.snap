// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputComponent Should match snapshot 1`] = `
<div
  className="root"
>
  <label
    htmlFor="input"
  >
    Type your message here: 
  </label>
  <input
    id="input"
    name="input"
    type="text"
  />
  <button
    id="button"
  >
    Send
  </button>
</div>
`;
